src/colophon.adoc

```diff
@@ -13,43 +14,94 @@ The document contains the following versions of the RISC-V ISA modules:
 [%autowidth,float="center",align="center",cols="^,<,^",options="header"]
 |===
 |Base |Version |Status
-|*RVWMO* |*2.0* |*Ratified*
 |*RV32I* |*2.1* |*Ratified*
-|*RV64I* |*2.1* |*Ratified*
 |*RV32E* |*2.0* |*Ratified*
 |*RV64E* |*2.0* |*Ratified*
+|*RV64I* |*2.1* |*Ratified*
 |_RV128I_ |_1.7_ |_Draft_
 
 h|Extension h|Version h|Status
 
+|*Zifencei* |*2.0* |*Ratified*
+|*Zicsr* |*2.0* |*Ratified*
+|*Zicntr* |*2.0* |*Ratified*
+|*Zihintntl* |*1.0* |*Ratified*
+|*Zihintpause* |*2.0* |*Ratified*
+|_Zimop_ | _0.1_ | _Draft_
+|*Zicond* | *1.0* |*Ratified*
 |*M* |*2.0* |*Ratified*
+|*Zmmul* |*1.0* |*Ratified*
 |*A* |*2.1* |*Ratified*
+|*Zawrs* |*1.01* |*Ratified*
+|*Zacas* |*1.0* |*Ratifed*
+|*RVWMO* |*2.0* |*Ratified*
+|*Ztso* |*1.0* |*Ratified*
+|*CMO* |*1.0* |*Ratified*
 |*F* |*2.2* |*Ratified*
 |*D* |*2.2* |*Ratified*
 |*Q* |*2.2* |*Ratified*
-|*C* |*2.0* |*Ratified*
-|_Counters_ |*2.0* |*Ratified*
-|_P_ |_0.2_ |_Draft_
-|*V* |*1.0* |*Ratified*
-|*Zicsr* |*2.0* |*Ratified*
-|*Zifencei* |*2.0* |*Ratified*
-|*Zihintpause* |*2.0* |*Ratified*
-|*Zihintntl* |*1.0* |*Ratified*
-|*Zfa* |*1.0* |*Ratified*
 |*Zfh* |*1.0* |*Ratified*
 |*Zfhmin* |*1.0* |*Ratified*
+|*Zfa* |*1.0* |*Ratified*
 |*Zfinx* |*1.0* |*Ratified*
 |*Zdinx* |*1.0* |*Ratified*
 |*Zhinx* |*1.0* |*Ratified*
 |*Zhinxmin* |*1.0* |*Ratified*
-|*Zmmul* |*1.0* |*Ratified*
-|*Ztso* |*1.0* |*Ratified*
+|*C* |*2.0* |*Ratified*
+|*Zc* |*1.0* |*Ratified*
+|*B* |*1.0* |*Ratified*
+|_P_ |_0.2_ |_Draft_
+|*V* |*1.0* |*Ratified*
+|*Zbk,Zk* |*1.0* |*Ratified*
+|*Zvb,Zvk* |*1.0* |*Ratified*
```

Comment:
LEt's stick to one extension per line.  Also, I don't think Zbk, Zvb, or Zvk is actually the name of an ISA extension.  I think a sufficient list is Zbkb, Zbkc, Zbkx, Zvbb, Zvbc, Zvkg, Zvkned, Zvknhb, Zvksed, Zvksh, Zvkt.

@nibrunieAtSi5 can you double-check my work?  Note I'm not trying to list out each option; I'm just trying to provide a mininum cover.  So I included e.g. Zvbb but not Zvkb, and I didn't include the NIST/SM superset extensions.

Note, `Zk` is correct as-is; it's just the other three that aren't.

Reply:
@aswaterman , your work for vector crypto LGTM

I guess you listed `Zvknhb` as a superset over `Zvknha` as well.

From what I can tell `Zk` does not contain `Zks`, so we may need to list `Zks`:

![image](https://github.com/riscv/riscv-isa-manual/assets/82109999/86dd5e16-97b5-4652-bd01-bed163078ace)

Reply:
Thanks.  @wmat the final list to replace these two lines as individual lines is: Zbkb, Zbkc, Zbkx, Zk, Zks, Zvbb, Zvbc, Zvkg, Zvkned, Zvknhb, Zvksed, Zvksh, Zvkt

Reply:
Done. Pushing change shortly.